app/controllers/products_controller.rb

```diff
@@ -305,35 +279,68 @@ def launch
 
   # private
 
-  def create_product_chat(product)
-    main_thread = product.discussions.create!(title: Discussion::MAIN_TITLE, user: current_user, number: 0)
-    product.update(main_thread: main_thread)
-    product.chat_rooms.create!(wip: main_thread, slug: product.slug)
+  def setup_core_team(product)
+    core_team_ids = Array(params[:core_team])
+    core_team_members = User.where(id: core_team_ids.select(&:uuid?))
+    core_team_members.each do |user|
+      product.core_team_memberships.create(user: user)
+    end
   end
 
-  def create_product_with_params
+  def find_the_chosen
+    chosen_ids = params[:product][:partner_ids] || ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.